demo/billing/resources/views/check_sink.sql.in

```diff
@@ -0,0 +1,14 @@
+CREATE MATERIALIZED VIEW check_sink AS
+SELECT
+reingested_sink.execution_time_ms AS t1,
+reingested_sink.monthly_bill AS bill1,
+billing_monthly_statement.execution_time_ms AS t2,
+billing_monthly_statement.monthly_bill AS bill2
+FROM
+reingested_sink, billing_monthly_statement
+
+WHERE
+to_timestamp(reingested_sink.month / 1000000) = billing_monthly_statement.month AND
```

Comment:
why do we have to do this conversion here?

Reply:
because we when we write down ScalarType::Timestamp as Avro we use the Avro `time-micros` logical type which stores the microseconds from epoch in a Avro i64, but then when we read the avro back in it just shows as a i64 without the annotation :(

Reply:
not sure if theres a quick fix but I think there is an issue that tracks this